tests/functionnal/index.js

```diff
@@ -0,0 +1,43 @@
+const {expect} = require('chai');
```

Comment:
The `tests/functionnal` folder name is a typo. Should be `tests/functional`

Reply:
Applied !